org.uqbar.project.wollok.lib/src/wollok/lib/WgameObject.xtend

```diff
@@ -73,6 +77,12 @@ class WgameObject {
 		board.addListener(listener)
 	}
 	
+	def findVisual(Gameboard it, WollokObject visual) {
+		it.components
+		.map[it as WVisual]
+		.findFirst[ it.wObject.equals(visual)]
```

Comment:
Not necessary to write "it" here in 2 places:  

it.components => components
it.wObject.equals(visual)  => wObject.equals(visual)